Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -160,14 +195,19 @@ extension AdjacencyList: VertexListGraph {
 		public var count: Int { vertexCount }
 	}
 
+	/// The collection of vertex identifiers.
```

Comment:
```suggestion
	/// The identifiers of all vertices.
```

There are lots of possible collections of vertex identifiers; this one is all of them.